internal/providers/main.go

```diff
@@ -1,3 +1,29 @@
 package providers
 
+import (
+	"errors"
+	"github.com/datadog/stratus-red-team/pkg/stratus"
+)
+
 const StratusUserAgent = "stratus-red-team"
+
+// EnsureAuthenticated ensures that the current user is properly authenticated against a specific platform
+func EnsureAuthenticated(platform stratus.Platform) error {
+	switch platform {
+	case stratus.AWS:
+		if !AWS().IsAuthenticatedAgainstAWS() {
+			return errors.New("you are not authenticated against AWS, or you have not set your region. " +
+				"Make sure you are authenticated against AWS, and you have a default region set in your AWS config " +
```

Comment:
This is more of an user experience, but what if we defaulted to `us-east-1` region? This way stratus would work even if no region was set, so only the authentication would need to be a problem that user has to fix (I am aware that setting region is quite easy though and maybe we want be explicit there).

Not a blocker from my side, just a thought.

Reply:
It's a good point that I actually did consider in the past. I chose to enforce having a region to follow the usage model of the AWS CLI, which crashes if you don't pass it a region